alpine-make-vm-image

```diff
@@ -334,7 +334,7 @@ while [ $# -gt 0 ]; do
 		-t | --serial-console) SERIAL_CONSOLE='yes'; n=1;;
 		-c | --script-chroot) SCRIPT_CHROOT='yes'; n=1;;
 		-h | --help) help 0;;
-		-V | --version) echo "$PROGNAME $VERSION"; exit 0;;
+		-V | --version) echo "$PROGNAME $VERSION"; return;;
```

Comment:
This is invalid usage of `return`:

> The return utility shall cause the shell to stop executing the current function or dot script. If the shell is not currently executing a function or dot script, the results are unspecified. [[source](https://pubs.opengroup.org/onlinepubs/009695399/utilities/return.html)]

Reply:
Maybe but it works and I think that it is only option to make it working without splitting to two separate files. Anyway will close this one then...

Reply:
> Maybe but it works

Just because it works in your shell doesn’t mean it works in all POSIX compliant shells.

```
bash: return: can only `return' from a function or sourced script
```

> I think that it is only option to make it working without splitting to two separate files.

Nope, this really isn’t the only option, it’s a pretty bad option.